op-node/rollup/driver/state.go

```diff
@@ -381,18 +373,41 @@ func (s *Driver) ResetDerivationPipeline(ctx context.Context) error {
 	}
 }
 
+func (s *Driver) syncStatus() *eth.SyncStatus {
```

Comment:
doc comment - particularly around invariants like should the event loop be blocked